phpini/phpini-lib.pl

```diff
@@ -308,6 +308,15 @@ sub graceful_apache_restart
 		&virtual_server::pop_all_print();
 		}
 	}
+if ($file && &get_config_fmt($file) eq "ini" &&
+	&foreign_installed("virtual-server") && 
+	&foreign_installed("virtualmin-nginx")) {
+	&foreign_require("virtual-server");
+	&foreign_require("virtualmin-nginx", "virtual_feature.pl");
+	my @dom = grep { &is_under_directory($_->{'home'}, $file) } 
+	              &virtual_server::list_domains();
+	&virtualmin_nginx::feature_restart_web_php($dom[0]);
```

Comment:
What if @doms is empty?

Reply:
> What if @doms is empty?

It really shouldn't be the case, but okay, sure i checked in a fix for that.

Reply:
Thanks!